WordPress/Classes/Services/AccountService.h

```diff
@@ -126,15 +126,6 @@ extern NSNotificationName const WPAccountEmailAndDefaultBlogUpdatedNotification;
  */
 - (void)restoreDisassociatedAccountIfNecessary;
 
-///--------------------
-/// @name Visible blogs
-///--------------------
-
-/**
- Sets the visibility for the given blogs
- */
-- (void)setVisibility:(BOOL)visible forBlogs:(NSArray *)blogs;
```

Comment:
Not sure if I missed any P2 posts, but removing the capability to hide sites seems to be a big change. Any chance we can preserve this feature in the new blog list?

Reply:
This feature was intentionally removed as part of the redesign. It also doesn't exist on wp.com.

Reply:
Can you remove the `Blog.visible` property too, since it now can only be `true` for all blogs? It's used by a few placed to exclude locally hidden blogs.

Reply:
Just confirming that this is the case – since it's not on WP.com it's likely to result in more support requests than it resolves?

FWIW I'd be ok to leave the Core Data property in place with a comment saying not to use it rather than introduce a new migration 🤷

Reply:
> likely to result in more support requests than it resolves?

I'm not sure I follow. I wasn't part of the original discussions about removal of this feature, but I 100% with it because this feature makes no sense, and it was, evidently, removed or never present on wp.com in the first place.

> FWIW I'd be ok to leave the Core Data property in place with a comment saying not to use it rather than introduce a new migration

I agree with Jeremy on this one. I know the app has been adding migrations nearly every release, which I think is a poor practice. I will add a deprecation message.

I found one more place where this property is used, and I will remove it.